generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html

```diff
@@ -49,9 +49,9 @@ <h1 jhiTranslate="login.title">Sign in</h1>
         <%_ if (enableSocialSignIn) { _%>
         <div class="col-md-6">
             <br/>
-            <jh-social provider="google"></jh-social>
-            <jh-social provider="facebook"></jh-social>
-            <jh-social provider="twitter"></jh-social>
+            <jhi-social provider="google"></jhi-social>
```

Comment:
`jhi` should be `<%= jhiPrefix %>` to match [the component selector](https://github.com/nkolosnjaji/generator-jhipster/blob/0c1820a1de30af6a721c70ea02b31b174322fab3/generators/client/templates/angular/src/main/webapp/app/shared/social/_social.component.ts#L7)